### ISSUE TITLE
Catch and display XMLSyntaxErrors

### ISSUE THREAD

**Issue by ajrbyers** [BirkbeckCTP/typesetting#186]
When a user previews an article from the Editor/Typesetter view we should catch and display XMLSyntaxError. From the author side we should ask them to report this issue to the editor.

**Comment by joemull** [comment]
Yes, let's do this when we do https://github.com/BirkbeckCTP/janeway/issues/3180
